subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/reservation/client/kubernetes/KubernetesReservationClient.kt

```diff
@@ -398,35 +395,32 @@ class KubernetesReservationClient(
         count: Int
     ) {
         val isDeploymentDone = waitForCondition(
-            logger = { logger.info(it) },
+            logger = { logger.debug(it) },
             conditionName = "Deployment $deploymentName deployed"
         ) {
             podsFromDeployment(
                 deploymentName = deploymentName
             ).size == count
         }
         if (!isDeploymentDone) {
-            val error =
-                RuntimeException("Something went wrong during deploying deployment: $deploymentName")
-            logger.critical(error.message.orEmpty())
-            throw error
+            throw RuntimeException("Can't create deployment: $deploymentName")
         }
     }
 
     private fun podsFromDeployment(
         deploymentName: String
     ): List<Pod> = try {
-        logger.info("Getting pods for deployment: $deploymentName")
+        logger.debug("Getting pods for deployment: $deploymentName")
         val items = kubernetesClient.pods().withLabel("deploymentName", deploymentName).list().items
         val runningPods = items.filter { it.status.phase == POD_STATUS_RUNNING }
-        logger.info(
+        logger.debug(
             "Getting pods for deployment: $deploymentName completed. " +
                 "Received ${items.size} pods (running: ${runningPods.size})."
         )
 
         items
     } catch (t: Throwable) {
-        logger.info("Failed to get pods for deployment: $deploymentName", t)
+        logger.warn("Failed to get pods for deployment: $deploymentName", t)
```

Comment:
I don't know is it a common way to get an exception?

Reply:
It may be a one time glitch. We need a retry for similar queries to k8s.